example/sparse_advection/sparse_advection_driver.cpp

```diff
@@ -122,12 +122,12 @@ TaskCollection SparseAdvectionDriver::MakeTaskCollection(BlockList_t &blocks,
                              mdudt.get(), beta * dt, mc1.get());
 
     // do boundary exchange
-    auto restrict =
+    auto restrict_task =
```

Comment:
seems totally reasonable to change this

Reply:
This var name was changed on develop, so I don't think this is an issue anymore.